dashboards-observability/public/components/event_analytics/utils/utils.tsx

```diff
@@ -359,3 +364,13 @@ export const filterDataConfigParameter = (parameter: ConfigListEntry[]) =>
   parameter.filter((configItem: ConfigListEntry) => configItem.label);
 
 export const getRoundOf = (value: number, places: number) => value.toFixed(places);
+
+export const getTooltipHoverInfo = ({ tooltipMode, tooltipText }: GetTooltipHoverInfoType) => {
+  if (tooltipMode === 'hidden') {
+    return 'none';
+  } else if (tooltipText === undefined) {
+    return 'all';
+  } else {
```

Comment:
similar here, it may not need this else block, just a `return tooltipText;` is enough